content/docs/reference/apis/_index.en.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "APIs"
+linkTitle: "APIs"
+description: "Programming interfaces specifications"
+weight: 60
+---
+
+RailJSON is the format used to describe a railway infrastructure, it's
```

Comment:
From the rendering, I would add some emphasis on the RailJSON, to make it equally important with openapis.
```suggestion
**RailJSON** is the format used to describe a railway infrastructure, it's
```

Reply:
Indee, done!